scripts/release-prompts.ts

```diff
@@ -126,10 +126,15 @@ export async function promptRelease(opts: BuildOptions): Promise<NormalizedRelea
 
   try {
     const answers = await inquirer.prompt<ReleasePromptAnswers>(prompts);
+    const npmTag = determineAnsweredTagToUse(answers);
+
+    if (npmTag === null) {
+      throw 'no proper tag found!';
```

Comment:
Any reason we wouldn't throw an error?
```suggestion
      throw new Error('no proper tag found!');
```

Reply:
oh yeah meant to do that! forgot to circle back